plotter/heat.go

```diff
@@ -223,3 +224,58 @@ func (h *HeatMap) GlyphBoxes(plt *plot.Plot) []plot.GlyphBox {
 	}
 	return b
 }
+
+// RasterHeatMap implements the plotter interface. It is similar to HeatMap,
+// apart from that the plot method uses a raster-based method when drawing
+type RasterHeatMap struct {
+	HeatMap *HeatMap
+}
+
+// NewRasterHeatMap creates as new heat map plotter for the given data,
+// using the provided palette.
+func NewRasterHeatMap(g GridXYZ, p palette.Palette) *RasterHeatMap {
+	return &RasterHeatMap{
+		HeatMap: NewHeatMap(g, p),
+	}
+}
```

Comment:
ISTM that this implementation would be better as an option on `HeatMap`. If we have a `Rasterized bool` field then the implementation can be easily chosen by the user without growing the API surface as significantly as is done here.

Reply:
I agree with this choice, I will incorporate the changes